### DISCUSSION TITLE
One Twilio Theme

### DISCUSSION THREAD

**Discussion by amyodea** [twilio-labs/paste#2981] [Q&A] [answered]
Hi Team 👋🏼 

I wanted to confirm the correct theme to be using in code for the new One Twilio design language is `twilio`?  

Thank you!

**Comment by gloriliale** [comment, marked as answer]
Hi @amyodea! Yes, that is the name of the new theme for the One Twilio design language, with `twilio-dark` being the dark theme for it. Our engineering team is currently updating the code components to be able to use them seamlessly with both, and we will release a version that automatically uses the `twilio` theme then.

**Reply by amyodea** [reply]
Amazing, thank you @gloriliale 👍
